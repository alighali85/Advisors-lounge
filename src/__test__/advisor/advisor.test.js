import React from 'react'
import { render } from '@testing-library/react'
import Advisor from '../../containers/advisor'


const AdvisorProps = {
    id: 1,
    name: 'testing_name',
    title: 'testing_title',
    bio: 'testing_bio',
    rating: 3
}

it('render the advisor card', () => {
    const { asFragment } = render(<Advisor {...AdvisorProps} />)
    expect(asFragment()).toMatchSnapshot()
})

it('has right title', () => {
    const { getByText } = render(<Advisor {...AdvisorProps} />)
    expect(getByText(AdvisorProps.title)).toHaveTextContent(AdvisorProps.title)
})