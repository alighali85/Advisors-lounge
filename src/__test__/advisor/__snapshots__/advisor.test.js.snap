// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has right title 1`] = `
<DocumentFragment>
  <div>
    <div
      class="card"
      style="width: 18rem;"
    >
      <div
        class="card-body"
      >
        <div
          class="card-title h5"
        >
          testing_name
        </div>
        <p
          class="card-text"
        >
          <i>
            testing_bio
          </i>
          <span>
            testing_title
          </span>
          <br />
          <span
            style="display: inline-block; direction: ltr;"
          >
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 0%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 0%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
          </span>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`render the advisor card 1`] = `
<DocumentFragment>
  <div>
    <div
      class="card"
      style="width: 18rem;"
    >
      <div
        class="card-body"
      >
        <div
          class="card-title h5"
        >
          testing_name
        </div>
        <p
          class="card-text"
        >
          <i>
            testing_bio
          </i>
          <span>
            testing_title
          </span>
          <br />
          <span
            style="display: inline-block; direction: ltr;"
          >
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 0%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 0%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
          </span>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`render the catd 1`] = `
<DocumentFragment>
  <div>
    <div
      class="card"
      style="width: 18rem;"
    >
      <div
        class="card-body"
      >
        <div
          class="card-title h5"
        >
          testing_name
        </div>
        <p
          class="card-text"
        >
          <i>
            testing_bio
          </i>
          <span>
            testing_title
          </span>
          <br />
          <span
            style="display: inline-block; direction: ltr;"
          >
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 100%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 0%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
            <span
              style="cursor: pointer; display: inline-block; position: relative;"
            >
              <span
                style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: rgb(204, 204, 204);"
              />
              <span
                style="display: inline-block; position: absolute; overflow: hidden; top: 0px; left: 0px; width: 0%;"
              >
                <span
                  style="display: inline-block; border-radius: 50%; border: 5px double white; width: 30px; height: 30px; background-color: red;"
                />
              </span>
            </span>
          </span>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
